devtools/install_arc.sh

```diff
@@ -24,17 +53,17 @@ cd ARC
 export PYTHONPATH=$PYTHONPATH:$(pwd)
 echo "export PYTHONPATH=$PYTHONPATH:$(pwd)" >> ~/.bashrc
 if { conda env list | grep 'arc_env'; } >/dev/null 2>&1; then
-    mamba env update -n arc_env -f environment.yml
+    $COMMAND_PKG env update -n arc_env -f environment.yml
 else
-    mamba env create -f environment.yml
+    $COMMAND_PKG env create -f environment.yml
 fi;
-mamba activate arc_env
+conda activate arc_env
 #make install-all
-###Temp###
-mamba create -n xtb_env python=3.7 -y
+###Temp### To be used until ARC repo install is properly updayed
```

Comment:
Can you open an issue on ARC if it needs to be updated?
(also, small typo here)

Reply:
Opened: https://github.com/ReactionMechanismGenerator/ARC/issues/580